progressbar_test.go

```diff
@@ -70,6 +90,16 @@ func ExampleFinish2() {
 	// 100% |██████████|  [0s:0s]
 }
 
+func ExampleXOutOfY() {
+	bar := New(100)
+	bar.SetPredictTime(false)
+
+	for i := 0; i < 100; i++ {
+		bar.Add(1)
+		time.Sleep(1 * time.Millisecond)
+	}
```

Comment:
Can you add the output here? Maybe change `i<100` to `i<50` and get the output and paste it on

```
// Output:
// 100% |█████     |  [50:100]
```

or whatever its supposed to be

Reply:
Yeah sure. It should only take a minute or two.

Reply:
Done.